lib/convertkit/client/subscribers.rb

```diff
@@ -14,6 +14,10 @@ def unsubscribe(email)
           f.params['email'] = email
         end
       end
+
+      def remove_tag_from_subscriber(subscriber_id, tag_id)
```

Comment:
@rkcudjoe What do you think about using keyword arguments? I would actually prefer to use them, but it would be inconsistent with the rest of the gem:

```ruby
def remove_tag_from_subscriber(subscriber_id:, tag_id:)
end
```

Reply:
@mrhead I don't mind using them if you think it's more appropriate for an API library; I'll just open an issue so that I can go back and update there rest to make it consistent across the board.